docs(aws): Fixing formatting, minor changes (#110)
spinnaker/spinnaker.io: content/en/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates.md

title: "AWS EC2 Launch Templates"
linkTitle: "AWS EC2 Launch Templates"
description: "AWS EC2 Launch Template Features, Use Cases and Sample API Requests"
weight: 2
---

> Launch Template Support must be enabled in order to use the features described in this page. 
> Follow guidelines [here](/docs/setup/other_config/server-group-launch-settings/aws-ec2/launch-templates-setup) to enable the feature set.

> > Note: The features supported in Deck come with helpful tool tips that aid in quick learning of the new features. 

Reviewer: ```suggestion
> Note: The features supported in the UI (Deck) come with tool tips that can help you learn the new features. 
```

  </tbody>
</table>

### Additional Features

> Enabling launch template support also unlocks additional features offered by Amazon EC2 Auto Scaling, that facilitate diversified server groups:

Reviewer: ```suggestion
Enabling launch template support also unlocks additional features offered by Amazon EC2 Auto Scaling that facilitate diversified server groups:
```
